File: uri 1036.py
import math
values = input().split(" ")
A, B, C = values
A, B, C = float(A), float(B), float(C)
cons = B**2 - 4 * A * C
if A != 0 and cons >= 0 :
    root1 = (-B + math.sqrt(cons)) / (2 * A)
    root2 = (-B - math.sqrt(cons))/ (2 * A)
    print("R1 = %.5f" % root1)
    print("R2 = %.5f" % root2)
else:
    print('Impossivel calcular')

File: uri 1036.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>


int main()
{
    float A,B,C,root1,root2,temp;
    scanf("%f %f %f",&A,&B,&C);
    temp = (B*B) - (4*A*C);

    if((A!=0) && (temp>=0)){
        root1 = ( -B+sqrt(temp))/(2*A);
        root2 = ( -B-sqrt(temp))/(2*A);
        printf("R1 = %.5f\nR2 = %.5f\n",root1,root2);
    }
    else{
        printf("Impossivel calcular\n");
    }

    return 0;
}
